src/main/java/com/algolia/search/models/indexing/Query.java

```diff
@@ -521,6 +521,15 @@ public Query setEnablePersonalization(Boolean enablePersonalization) {
     return this;
   }
 
+  public Integer getPersonalizationImpact() {
```

Comment:
blargh this change is completely unrelated to the multi-module thing :) you should push that in another PR

Reply:
arg I checked out too fast 😞